templates/landscape/pricing.html

```diff
@@ -88,38 +88,50 @@ <h1 class="u-sv4">Choose the best Landscape for you:</h1>
           <td></td>
           <td class="u-align--center"><i class="p-icon--success"></i></td>
         </tr>
-      </tbody>
-    </table>
-  </div>
-  <div class="u-fixed-width">
-    <table class="p-table">
-      <thead>
-        <tr>
-          <th style="font-size: 20px;">Product pricing</th>
-          <th class="u-align--center" style="font-size: 20px;">Self hosted Landscape</th>
-          <th class="u-align--center" style="font-size: 20px;">Landscape SaaS</th>
-        </tr>
-      </thead>
-      <tbody>
         <tr>
-          <td></td>
-          <td class="u-align--center">Free for up to 10 machines for personal use, or evaluation purposes</td>
-          <td class="u-align--center"><a href="/pricing">See pricing</a></td>
+          <td>Pricing</td>
+          <td><a href="/advantage/subscribe">Pricing calculator for Ubuntu Advantage</a></td>
+          <td>Free for up to 10 machines for personal use, or evaluation purposes.<br /><br />Beyond the free tier, <a href="/advantage">Ubuntu Advantage</a> is required for more than 10 machines on self-hosted Landscape</td>
         </tr>
         <tr>
           <td></td>
-          <td class="u-align--center">UA subscription required for more than 10 machines</td>
-          <td class="u-align--center"><a href="/pricing">See pricing</a></td>
-        </tr>
-        <tr>
-          <td></td>
-          <td class="u-align--center"><a href="/landscape/install">Set up your self hosted Landscape</a></td>
-          <td class="u-align--center"><a href="/support">Learn more about Ubuntu Advantage</a></td>
+          <td>Landscape SaaS is included with <a href="/advantage">Ubuntu Advantage</a></td>
+          <td><a href="/landscape/install">Set up your self-hosted Landscape</a></td>
         </tr>
       </tbody>
     </table>
   </div>
 </section>
 
+<section class="p-strip--light">
+  <div class="row u-equal-height">
+    <div class="col-6 u-vertically-center">
+      <div>
+        <h2>Speak to a member of our team</h2>
+        <p>If you want to learn more about Landscape or our professional services options, contact us to discuss your needs.</p>
+        <p><a href="#get-in-touch" class="p-button--positive js-invoke-modal">Contact us</a></p>
```

Comment:
sorry one more thing if it's better! 
```suggestion
        <p><a href="/landscape/pricing#get-in-touch" class="p-button--positive js-invoke-modal">Contact us</a></p>
```